src/sections/General/Footer/footer.style.js

```diff
@@ -98,6 +98,7 @@ const FooterWrapper = styled.section`
 			border: 1px solid #FFFFFF;
 			background-color: #000000;
 			border-radius: 0.25rem;
+			margin-bottom:14px;
```

Comment:
Let's limit this proprty to mobile view only, and update the value to `0.75rem` instead of using `px`

Reply:
This is still not implemented correctly.
You will need to use a media query to address this comment.